### PR TITLE
Add Tetris Ultimate

### PR DESCRIPTION
**Pull request by Jeremiidesu**
Adds Tetris Ultimate.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/tetris-ultimate/game.dat [hunk 1]
```diff
@@ -0,0 +1,14 @@
+title = "Tetris Ultimate"
+description = "Tetris Ultimate is a 3DS release of the famous puzzle game, developed and published by Ubisoft Osaka, featuring classic modes such as Marathon but also new ones such as Ultimate Duel."
```
Comment (CodingKoopa) [inline games/tetris-ultimate/game.dat]: This is a bit run-on, I would split this up into two sentences, e.g. "game, developed and published by Ubisoft Osaka. It features classic..."

File: games/tetris-ultimate/game.dat [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+needs_system_files = true
+needs_shared_font = false
+
+[[releases]]
+title = "0004000000132400"
```
Comment (CodingKoopa) [inline games/tetris-ultimate/game.dat]: USA release first.

File: games/tetris-ultimate/game.dat (hunk 1 above)
Comment (Jeremiidesu) [reply games/tetris-ultimate/game.dat]: Done

File: games/tetris-ultimate/game.dat (hunk 2 above)
Comment (Jeremiidesu) [reply games/tetris-ultimate/game.dat]: Done

**Comment by Jeremiidesu** [comment]
Fixed the mentioned issued and added a third screenshot; I guess that'll do?

File: games/tetris-ultimate/game.dat [hunk 3]
```diff
@@ -0,0 +1,14 @@
+title = "Tetris Ultimate"
+description = "Tetris Ultimate is a 3DS release of the famous puzzle game, developed and published by Ubisoft Osaka. It features classic modes such as Marathon but also new ones such as Ultimate Duel."
```
Comment (CodingKoopa) [inline games/tetris-ultimate/game.dat]: Rephrase the last sentence: "It features classic modes such as Marathon, and also..."

**Comment by CodingKoopa** [comment]
>Fixed the mentioned issues and added a third screenshot; I guess that'll do?

We usually require 2 reviews, so when someone else LGTMs this PR, then we can merge it.

File: games/tetris-ultimate/game.dat (hunk 3 above)
Comment (Jeremiidesu) [reply games/tetris-ultimate/game.dat]: Done

**Comment by Jeremiidesu** [comment]
Huh, no idea how it went through. I'll go and update all three screenshots so they show an english version of the game, i think that'll be better

**Comment by Jeremiidesu** [comment]
Reuploaded screenshots, guaranteed cursor-free and in English. Updated game.dat according to the actual english name of the Battle Ultimate mode.

**Pull request merged**